Dt-42-44-45

aimsngn opened the pull request:
This code contains components (ts, html, css) for the overview page, treatment plan page, and health records page. (did not make a nav bar for it though).

mrrhetorical commented on pro-dental-care-ui/src/app/patient/overview/patient-overview.component.html:
```diff
@@ -0,0 +1,49 @@
+      <!-- Appointment -->
+      <article class="overview-box">
+        <div class="icon-placeholder" aria-label="Appointment illustration">
+          <img src="assets/clock.png" alt="Appointment Icon" />
```
In this and other places where you use images sources, please use the `ngSrc` attribute instead of src.
Also, for the source of the image you should use full paths such as `/assets/clock.png` with the leading forward slash.

aimsngn replied:
fixed it! if its all good, u can merge pull request! :)

mrrhetorical reviewed (changes requested):
Please update image `src` attributes to use `ngSrc` and use absolute paths. Otherwise,  everything looks great!

mrrhetorical commented:
Thank you for updating the `src` attributes to `ngSrc` attributes. The `ngSrc` is specified as `ngSrc=<uri>` instead of `[ngSrc]=<uri>`, however. You will also need to import the `NgOptimizedImage` component in each class you use it in.

aimsngn commented:
I added the ngSrc imports, and changed it to ngSrc=<uri>. 

However, for the acordians, since it needs evaluate expressions based on whether the acordian's expanded or not, we need to use [ngSrc]=<uri>.

Thanks! Tried it this time and it works!!
